plume-front/src/main.rs

```diff
@@ -1,56 +1,35 @@
 #![recursion_limit="128"]
+#![feature(decl_macro, proc_macro_hygiene)]
+
+extern crate gettext;
+#[macro_use]
+extern crate gettext_macros;
+#[macro_use]
+extern crate lazy_static;
 #[macro_use]
 extern crate stdweb;
 
-use stdweb::{unstable::{TryFrom, TryInto}, web::{*, event::*}};
+use stdweb::{web::{*, event::*}};
+
+init_i18n!("plume-front", en, fr);
+
+mod editor;
+
+compile_i18n!();
+
+lazy_static! {
+    static ref CATALOG: gettext::Catalog = {
+        let catalogs = include_i18n!();
```

Comment:
I love that it just works like in the back-end, with nothing more to

Reply:
Yeah, that's really great! The only downside is that we include all the translations in the .wasm so it is a bit bigger, but there is like 3 or 4 messages for the moment so I guess it's fine (and I don't expect it to become much bigger, I think we will never have more than 30 messages for the front end).

Reply:
considering all Plume translations (not plume-front) are 208k on my computer, any picture loaded on the front page must cost much more than having all translations available

Reply:
so how big is the dynamic front-end right now? and how much of that is cachable?

Reply:
It is 2.2M on my computer, but there are various ways to optimize WASM size that we didn't explored yet. And I don't know how much of it is cachable? Nothing?

Reply:
I'd say everything, it's a static file after all. I tried compiling with `cargo web deploy --release`, it seems to work, and is "only" 832k. We could also try to compact plume-front.js as it's 35k with indentation and things, however I'm not sure it's that important